raiden-cli/src/raiden.ts

```diff
@@ -117,30 +117,47 @@ function parseArguments() {
     .help().argv;
 }
 
-async function askUserForPassword(address: string): Promise<string> {
-  const userInput = await inquirer.prompt<{ password: string }>([
-    { type: 'password', name: 'password', message: `[${address}] Password:`, mask: '*' },
-  ]);
-  return userInput.password;
+async function listAccounts(keystoreDir: string) {
```

Comment:
I'll rename and add the explicit return type, but in our codebase, we don't require it, and several functions use the implicit return type, I don't think we should enforce it, specially because some of them are quite complex and error prone to forget to update on long interface declarations which currently are implicit.